models/comments.model.js

```diff
@@ -24,3 +24,25 @@ exports.insertComment = (comment, article_id) => {
       return rows[0];
     });
 };
+
+exports.checkCommentExists = (comment_id) => {
+  return db
+    .query(`SELECT * FROM comments WHERE comment_id = $1`, [comment_id])
+    .then(({ rows }) => {
+      if (!rows.length) {
+        return Promise.reject({
+          status: 404,
+          msg: "Not found",
+        });
+      }
+    });
+};
+
+exports.removeComment = (comment_id) => {
+  return db
+    .query(
+      `DELETE FROM comments
+      WHERE comment_id = $1`,
+      [comment_id]
+    )
+};
```

Comment:
This can be done in a single query. 

Hint: How might you get a value back from your delete query to check what has been deleted?